.github/workflows/autodeploy.yaml

```diff
@@ -0,0 +1,26 @@
+name: Deploy Storybook to S3
+
+on:
+  push:
```

Comment:
```suggestion
on:
  pull_request:
  push:
```

you can add a `pull_request:` trigger to test on this PR

Reply:
ah nice, thanks